test/api/importcsv/api_test.go

```diff
@@ -0,0 +1,78 @@
+package importcsv
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestImportCSV(t *testing.T) {
+	for _, r := range TestCases {
+		t.Run(r.FileName, func(t *testing.T) {
+
+			// Upload CSV.
+			inputData := api.ImportSummary{}
+			assert.Must(t, Client.FilePost(api.SummariesRoot+"/upload", r.FileName, &inputData))
```

Comment:
This is not the right way creating the path, please follow what you had in your previous PR https://github.com/konveyor/tackle2-hub/pull/375/files#diff-6ded4e5e60480daa4db7cf0740809d6b470d937e85d1b674b02777560fed1a38R25